pytorch_lightning/accelerators/gpu.py

```diff
@@ -79,6 +80,10 @@ def auto_device_count() -> int:
         """Get the devices when set to auto."""
         return torch.cuda.device_count()
 
+    @staticmethod
+    def is_available() -> bool:
+        return torch.cuda.is_available() and torch.cuda.device_count() > 0
```

Comment:
```suggestion
        return torch.cuda.is_available() and torch.cuda.device_count() > 0
```
 Isn't that redundant? Doesn't `torch.cuda.is_available()` only return `True` if there are also some devices?

Reply:
you're right, this is redundant. device_count internally calls `is_available`